pkgs/by-name/ka/kaon/package.nix

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: 2025 coolGi <me@coolgi.dev>
+#
+# SPDX-License-Identifier: MIT
+{
+  lib,
+  stdenv,
+  qt6,
+  cmake,
+
+  xrSources,
+}:
+let
+  ValveFileVDF_src = builtins.fetchTarball {
+    url = "https://github.com/TinyTinni/ValveFileVDF/archive/refs/tags/v1.1.1.tar.gz";
+    sha256 = "sha256:1qg66p68fn8c2pifqkzz490b366dvcpq6j779b9k2c1ahwf35g5k";
+  };
+in
+stdenv.mkDerivation {
+  inherit (xrSources.kaon)
+    pname
+    version
+    src
+    date
+    ;
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtquick3d
+  ];
+  nativeBuildInputs = [
+    qt6.wrapQtAppsHook
+    cmake
+  ];
```

Comment:
nit:
I'd prfer this to be above buildInputs and have one empty line between this and buildInputs